packages/shared/src/components/Feed.tsx

```diff
@@ -408,6 +427,7 @@ export default function Feed<T>({
         actionButtons={actionButtons}
         isHorizontal={isHorizontal}
         feedContainerRef={feedContainerRef}
+        shouldShowSurvey={shouldShowSurvey && !isLaptop}
```

Comment:
Why is this condition and the condition above different? one `isLaptop` and the other `!isLaptop`?

Reply:
We have different placements for laptop and touch devices.

![Uploading Screenshot 2024-06-04 at 7.29.08 PM.png…]()